Add some guidelines to the instructions (#159)
cncf/presentations: README.md

Thanks for your patience!

### Submission Process

> This is inprogress and subject to change, please feel free to PR ideas and changes you'd like to see:

Reviewer: ```suggestion
This is a work in progress and subject to change. Feel free to submit Pull Request (PR) for ideas and changes you'd like to see.
```

> > What kind of talks are accepted v. Won’t be accepted?

Reviewer: ```suggestion
> What kind of talks are accepted vs. not accepted?
```

> For now we are concentrating on talks given at CNCF Events since those have already been accepted by the community. We don't want to do editorializing in this repo, if it's already been given at a CNCF event submit it. 

Reviewer: ```suggestion
We are currently focusing on talks presented at CNCF Events, as these have already been vetted and accepted by the community. We aim to avoid editorializing in this repository. If your talk has been presented at a CNCF event, feel free to submit it.
```

> > Do the talks have to be focused on cloud native projects only or can they submit talks about other open source talks?

Reviewer: ```suggestion
> Do the talks have to be focused solely on cloud native projects, or can they cover other open source topics?
```

> We are hoping to use this as a prototype so we can copy the pattern to include a wider range of topics, but for now we're trying to capture the low hanging CNCF fruit. In the meantime, if it's a good talk, submit it!

Reviewer: ```suggestion
While our initial focus is on capturing talks related to CNCF projects, we hope to expand the scope to include a broader range of topics in the future. If you believe your talk is valuable, go ahead and submit it!
```

> > Is there a limit on how many talks someone can submit?

Reviewer: ```suggestion
> Is there a limit on the number of talks one can submit?
```

> The intent is to gather as many CNCF talks as possible, so the limit is based on how many talks the presenter can actually deliver. :D 

Reviewer: ```suggestion
Our goal is to collect as many CNCF-related talks as possible. Therefore, the only limit is the number of talks you can realistically present. :D 
```

> > Can I submit on someone else's behalf?

Reviewer: ```suggestion
> Can I submit talks on someone else's behalf?
```

> Most of this information is public and published on a schedule. If you want to help your friends and colleagues submit their own talks around specific topics to help us grow that tag's content, then we encourage you to do so! Please check with the presenter before submitting a PR as a common courtesy.

Reviewer: ```suggestion
Most of the information related to these talks is publicly available and published on schedules. If you wish to help your friends and colleagues submit their talks to enrich specific topic tags, you're encouraged to do so. However, please consult with the presenter before submitting a PR as a common courtesy.
```
